src/containers/detail/Detail.tsx

```diff
@@ -38,47 +40,17 @@ export class Detail extends Component<RouteComponentProps<DetailProps>, DetailSt
-				lom_classification: [
-					'beeldende kunsten - schilderkunst, beeldhouwkunst, bouwkunst',
-					'fotografie - video - film',
-					'gedrags- en cultuurwetenschappen',
-					'muzikale opvoeding - muziek',
-					'PAV - MAVO - GASV - ASPV',
-				],
-				lom_typical_age_range: ['Secundair 2de graad', 'Secundair 3de graad'],
-				lom_intended_enduser_role: ['Docent', 'Student'],
-				duration_time: '00:08:32',
-				duration_seconds: 512,
-				administrative_type: 'video',
-				administrative_external_id: '8911n96442',
-			},
+			item: {},
 		};
 		// TODO: get item from store by id
-		// const itemId: string = (props.match.params as any)['id'];
+		const itemId: string = (props.match.params as any)['id'];
+		getDetail(itemId)
+			.then((detailResponse: DetailResponse) => {
+				this.setState({ item: detailResponse });
+			})
+			.catch(err => {
+				console.error('Failed to get detail frop the server', err, { id: itemId });
```

Comment:
remove console.error

Reply:
deze moeten vervangen worden door een Alert (toast) in de message stack, maar ik ben die component nog aan het schrijven. Voorlopig zou ik de console.errors laten staan, zodat we goed zien waar er nog alerts moeten komen